lvm/lvm-util.c

```diff
@@ -132,10 +132,7 @@ lvm_open_vg(const char *vgname, struct vg *vg)
 		goto out;
 	}
 
-	for (;;) {
-		if (lvm_read_line(scan))
-			break;
-
+	for (i = 0; !lvm_read_line(scan); ++i) {
```

Comment:
In principle, this modification is not exactly the same as the code before: it does not add elegance because
the previous code have 2 break conditions, while here one break condition is moved into the for.

Moreover, the code as it is now, works anyway but it is just because there is always a newline at the end
of the output but if the output were a single line without newlines, it would break at the very end of the
loop without ever having the chance of increasing "i".

I think it is more reasonable to leave the **for** statement as is and add an **else** clause the the first **if**, like
```
if (lvm_read_line(scan))
    break;
else
    i = 1;
```